aries-site/src/pages/templates/persistent-navigation.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+import { BulletedList, CardGrid, Meta, SubsectionText } from '../../components';
+import { MinimalSidebarExample } from '../../examples';
+import { ContentSection, Example, Layout, Subsection } from '../../layouts';
+import { getPageDetails, getRelatedContent } from '../../utils';
+
+const title = 'Persistent Navigation';
+const topic = 'Templates';
+const page = getPageDetails(title);
+const relatedContent = getRelatedContent(title);
+
+const Navigation = () => (
+  <Layout title={title}>
+    <Meta
+      title={title}
+      description={page.seoDescription}
+      canonicalUrl="https://design-system.hpe.design/templates/persistent-navigation"
+    />
+    <ContentSection>
+      <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example
+        // code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/dashboards/DashboardExample.js"
+        // template
+        >
+          {}
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Persistent navigation aids the user in orientation and provides
```

Comment:
Maybe:
Persistent navigations aids the user in maintaining context of where they are location within an application. This presence eases the user experience, especially in information rich applications.